#include <stdexcept>
#include "s-libexcept.hpp"

XXX::XXX() {
        throw std::runtime_error("XXX error");
}
